src/utilities.cpp

```diff
@@ -0,0 +1,41 @@
+// Copyright 2023 Brandon Matthews <thenewwazoo@optimaltour.us>
+// All rights reserved. GPLv3 License
+
+#include "utilities.h"
+#include "log.h"
+#include "LittleFS.h"
+
+
+void sync_and_restart() {
```

Comment:
This function calls the GDO Security+ 2.0 "sync" operation to mind. maybe "graceful_restart"?